rpm/archivematica/package.spec

```diff
@@ -0,0 +1,342 @@
+# Globals
+%global __os_install_post %(echo '%{__os_install_post}' | sed -e 's!/usr/lib[^[:space:]]*/brp-python-bytecompile[[:space:]].*$!!g')
+
+### Packages
+Name: archivematica
+Version: 1.4.1
+Release: 1
+Summary: Archivematica 
+Group: Application/System
+License: AGPLv3
+Source0: https://github.com/artefactual/archivematica/
+BuildRequires: git, gcc, openssl-devel, python-virtualenv, python-pip, mariadb-devel, libxslt-devel, python-devel
+Requires: archivematica-common
```

Comment:
Yes, it's a requirement for the installation, the ones needed for building are in the BuildRequires tag.